docs/Performance.md

```diff
@@ -1 +1,41 @@
 # Performance
+
+## Logs
+
+### CPU usage guidelines
+
+#### Setup
+
+The following benchmark has been compiled on an Amazon `m4.large`
+instance (which has 2 CPU cores and 8 GB of memory available).
+
+It can be used when estimating the required CPU resources for logs collection
+using [`filelogreceiver`][filelogreceiver].
+
+[filelogreceiver]: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver
+
+#### Benchmark
+
+Events Per Second (EPS) achieved for a particular average message size and CPU usage.
+
+|                       | 100B | 512B  | 1KB  | 5KB  | 10KB |
+|:---------------------:|:----:|:-----:|:----:|:----:|:----:|
+| **Average CPU usage** |  -   |   -   |  -   |  -   |  -   |
+|          5%           | 2000 | 1100  | 1000 | 150  | 200  |
+|          10%          | 3500 | 2100  | 1500 | 450  | 300  |
+|          20%          | 6500 | 4100  | 3000 | 1200 | 700  |
+|          50%          | 1400 | 10100 | 8500 |  -*  |  -*  |
+|          90%          |  -*  | 19100 |  -*  |  -*  |  -*  |
```

Comment:
| **Average CPU usage** \ **Average Message Size**                      | 100B | 512B  | 1KB  | 5KB  | 10KB |
|:---------------------:|:----:|:-----:|:----:|:----:|:----:|
|          5%           | 2000 | 1100  | 1000 | 150  | 200  |
|          10%          | 3500 | 2100  | 1500 | 450  | 300  |
|          20%          | 6500 | 4100  | 3000 | 1200 | 700  |
|          50%          | 1400 | 10100 | 8500 |  -*  |  -*  |
|          90%          |  -*  | 19100 |  -*  |  -*  |  -*  |

vs


|                       | 100B | 512B  | 1KB  | 5KB  | 10KB |
|:---------------------:|:----:|:-----:|:----:|:----:|:----:|
| **Average CPU usage** |  -   |   -   |  -   |  -   |  -   |
|          5%           | 2000 | 1100  | 1000 | 150  | 200  |
|          10%          | 3500 | 2100  | 1500 | 450  | 300  |
|          20%          | 6500 | 4100  | 3000 | 1200 | 700  |
|          50%          | 1400 | 10100 | 8500 |  -*  |  -*  |
|          90%          |  -*  | 19100 |  -*  |  -*  |  -*  |

vs


|                       | **Average Message Size**  | 100B | 512B  | 1KB  | 5KB  | 10KB |
|:---------------------:|:----:|:-----:|:----:|:----:|:----:|:----:|
| **Average CPU usage** |  -   |   -   |  -   |  -   |  -   | - |
|          5%           |   -   | 2000 | 1100  | 1000 | 150  | 200  |
|          10%          |   -   | 3500 | 2100  | 1500 | 450  | 300  |
|          20%          |   -   | 6500 | 4100  | 3000 | 1200 | 700  |
|          50%          |   -   | 1400 | 10100 | 8500 |  -*  |  -*  |
|          90%          |    -   | -*  | 19100 |  -*  |  -*  |  -*  |

I don't really like any of them but I think first is the most correct

Reply:
in addition `Average CPU usage` should be bolded

Reply:
I'd also say that the first is the most correct but I'd vote for second (as is) in terms of readability.

Reply:
sadly agree with that